src/beacon-api/app/routes/devices.py

```
@@ -505,4 +505,145 @@ async def get_offline_devices(
+    *,
+    db: Session = Depends(get_db),
+    days: int = Query(30, ge=1, le=365, description="Days ahead to check for upcoming maintenance")
+):
+    current_time = datetime.now(timezone.utc)
+    cutoff_time = current_time + timedelta(days=days)
+    
+    # Get devices with upcoming maintenance
+    devices = db.exec(
+        select(Device)
+        .where(
+            (Device.next_maintenance != None) & 
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)